internal/state/execution.go

```diff
@@ -434,14 +438,17 @@ func buildLastCommitInfo(block *types.Block, store Store, initialHeight int64) a
 
 func extendedCommitInfo(c abci.CommitInfo, votes []*types.Vote) abci.ExtendedCommitInfo {
 	vs := make([]abci.ExtendedVoteInfo, len(c.Votes))
+	var ext []byte
 	for i := range vs {
+		if c.Votes[i].SignedLastBlock {
+			ext = votes[i].Extension
```

Comment:
(related to a comment above) Are we sure the size of `votes` equals the size of `c.Votes` ?